source/MaterialXGenMdl/MdlSyntax.cpp

```diff
@@ -533,4 +555,41 @@ void MdlSyntax::makeValidName(string& name) const
     }
 }
 
+string MdlSyntax::modifyPortName(const string word) const
```

Comment:
Minor note: Should this be a const reference rather than a const value?

Reply:
the input parameter? yes,  that can/should be a `const string&`